src/daily_output.ts

```diff
@@ -133,6 +136,7 @@ export class DailyOutput {
 		});
 
 		this.data.ranks.daily.groupEffortRewards = [...top50Data, ...otherParticipantsData];
+		this.data.ranks.weekly.groupEffortRewards = [...top50Data, ...otherParticipantsData];
```

Comment:
I don't think we can do that... if we overwrite the weekly rank with daily data we will lose data. We need to merge with previous data.

Reply:
The daily and the weekly must always have the same data. Both refer to the same week instance.
If there's a new top `#1`, we must update that in the weekly as well